relays/ethereum/src/substrate_client.rs

```diff
@@ -187,9 +187,9 @@ impl SubstrateRpc for SubstrateRpcClient {
 
 	async fn grandpa_authorities_set(&self, block: Hash) -> Result<GrandpaAuthorityList> {
 		let call = SUB_API_GRANDPA_AUTHORITIES.to_string();
-		let data = Bytes(block.as_bytes().to_vec());
+		let data = Bytes(Vec::new());
 
-		let encoded_response = Substrate::state_call(&self.client, call, data, None).await?;
+		let encoded_response = Substrate::state_call(&self.client, call, data, Some(block)).await?;
```

Comment:
Just for posterity, why is `data` empty and the 4th parameter is set?

Reply:
Because block hash is the 4th argument of RPC method, not the runtime call - i.e. we call method `call` with arguments `data` at the state-od-block `Some(block)`.